### ISSUE TITLE
`unused binding` warning reported with `in-ns`

### ISSUE THREAD

**Issue by fjolne** [borkdude/clj-kondo#348]
**version**

clj-kondo v2019.07.05-alpha

**platform**

macOS Mojave 10.14

**problem**

clj-kondo reports `unused binding` warning if the binding is only used by `in-ns`.

**repro**

```clojure
(let [ns-name "user"]
  (in-ns ns-name))
```

**expected behavior**

No warning.

**actual behavior**

`warning: unused binding ns-name`

**Comment by borkdude** [comment]
Thanks!

**Comment by borkdude** [comment]
@fjolne Should be fixed with this binary: https://4582-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.07.06-alpha-SNAPSHOT-macos-amd64.zip

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/8a10eb23d8626fda1d5c877beb4bb0b767e4e6ab

**Comment by fjolne** [comment]
Great, thanks! I've been on my way to integrate clj-kondo to our team's workflow via pre-commit hooks, good to go now.
